packages/backend/src/docs/rest-api.json

```
@@ -1,1114 +1,1144 @@
-                {
-                    "name": "id",
-                    "in": "path",
+          },
+          "500": {
+            "$ref": "#/responses/500"
+          }
+        }
+      },
+      "parameters": [
+        {
+          "name": "username",
+          "in": "path",
+          "type": "string",
+          "required": true
+        }
+      ]
+    },
+    "/settings/refreshRate": {
+      "get": {
+        "tags": [
+          "Moodle"
+        ],
+        "summary": "Gets moodle fetch interval",
+        "description": "Returns the refresh interval in which the current data is requested and notifications are sent accordingly.",
+        "operationId": "getRefreshRateRequest",
+        "security": [
+          {
+            "Bearer": []
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation",
+            "schema": {
+              "$ref": "#/definitions/ApiResponse"
+            }
+          },
+          "400": {
+            "$ref": "#/responses/400"
+          },
+          "401": {
+            "$ref": "#/responses/401"
+          },
+          "500": {
+            "$ref": "#/responses/500"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Moodle"
+        ],
+        "summary": "Sets moodle fetch interval",
+        "description": "Sets the refresh interval in which the current dataset is requested and notifications are sent accordingly.",
+        "operationId": "setRefreshRateRequest",
+        "security": [
+          {
+            "Bearer": []
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "description": "New refresh rate in milliseconds",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "refreshRate": {
+                  "type": "number",
+                  "example": 60000,
+                  "minimum": 5000,
+                  "maximum": 2147483647
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation"
+          },
+          "400": {
+            "$ref": "#/responses/400"
+          },
+          "401": {
+            "$ref": "#/responses/401"
+          },
+          "500": {
+            "$ref": "#/responses/500"
+          }
+        }
+      }
+    },
+    "/settings/courses": {
+      "get": {
+        "tags": [
+          "Moodle"
+        ],
+        "summary": "Gets course list",
+        "description": "Returns a list of all courses with the status whether the associated notifications are enabled or disabled.",
```

```suggestion
        "description": "Returns a list of all courses including a status indicating whether the associated notifications are enabled or disabled.",
```
Overlooked this last time :smile: